### PR TITLE
Add links and guidance on the Mural notes template

### PR DESCRIPTION
**Pull request by andaleliz**
Adding references to Mural template for observer notes in the **Running Sessions** and **Synthesis** sections

### CODE REVIEW

File: teams/vsa/design/vsa-research-process.md [hunk 1]
```diff
@@ -91,6 +95,8 @@ The maximum given to you according to our contract to synthesize research is “
 
 The following process is recommended by the VSA Design Lead:
 1. At the end of each day of research, schedule a “debrief” session where you invite all the observers to share the key takeaways from the sessions. Keep track of these key takeaways.
+    * On the right side of the Mural template, there is a Top Observation section, which is handy during the debrief sessions.  A benefit of this is team cohesion and getting folks interested in the research
+    * Use To Observations to discuss "what is top of mind about participant X" with your observers? No need to get a super thorough synthesis, just what sticks out based on what we heard.
```
Comment (sshein) [inline teams/vsa/design/vsa-research-process.md]: this bullet is a little awkwardly worded?

File: teams/vsa/design/vsa-research-process.md [hunk 2]
```diff
@@ -78,6 +78,10 @@ We have to be careful with -- and if possible avoid using -- recordings because
    * Type “observer instructions” in the channel at the beginning of your day of sessions to give them tips
    * [Observer instructions can also be found here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/during-research/howto-observer-instructions.md)
    * At the end of your session, take a look at the feedback backchannel and see if anyone brought up questions that you can address with your participant
+* A [Mural template](https://app.mural.co/template/0e63994b-bce9-4b53-8e37-b5293a2b4dac/20381d25-a5d5-4457-9893-eeb2e9cb6ff4) is available to help observers take notes during your sessions.
+  * The template has a section for each participant, and within the participant section, you can create a section for each task. 
+  * Share the mural board with your team, a day in advance of starting any sessions. Ask them to write any observations they have on the mural board.  Usually there are maybe 1-2 people who will be actively writing things, and that’s okay.
```
Comment (sshein) [inline teams/vsa/design/vsa-research-process.md]: Maybe add here that we don't expect the moderator or the official "note taker" to be using the Mural board, but it's for any other observers that might want to put in notes during the sessions

**Review by sshein — CHANGES_REQUESTED** [review]
Added a few comments for changes inline. thanks!

File: teams/vsa/design/vsa-research-process.md (hunk 1 above)
Comment (andaleliz) [reply teams/vsa/design/vsa-research-process.md]: Ah yes - it was supposed to say **Top Observations**, not To observations. How about:

Use the Top Observation section to capture what your observers feel sticks out from each session. There is no need to get a super thorough synthesis at this stage.

**Comment by sshein** [comment]
resolving, you can make your edits directly!

**Pull request merged**